ansible/roles/products/teleop/control_machine/deploy/defaults/main.yml

```diff
@@ -27,15 +27,13 @@ user_folder: "/home/{{ user }}"
 shadow_hand_launcher_folder: "{{ user_folder }}/.shadow_launcher_app_{{ container_name }}/shadow_hand_launcher"
 setup_directory: "/usr/local/bin"
 use_aws: false
-launch_hand: false
 server_hostname: "SERVER"
```

Comment:
can this be not hardcoded?

Reply:
What do you mean? Im removing this as the functionality of how docker_deploy icons working is changing completely now

Reply:
This is just the default value. Its overrided by another role later on that updates the default:

`- name: Set server hostname
  set_fact:
    server_hostname: "{{ hostvars[groups['server'][0]]['ansible_host'] }}"
  when: groups['server'] is defined`